honeycombio/honeycomb-opentelemetry-python
# docs: Update example apps

## Which problem is this PR solving?

- Closes #78 

## Short description of the changes
- Added metrics example
- Added local visualizations to flask example
- Added all options available with descriptions to basic example
- Reorganized readmes a bit and added some more details to example READMEs

## How to verify that this has the expected result
- Configure a metrics dataset and run the flask app, verify that you see metrics in Honeycomb
- Run basic app and click on local vis link
